docs/theory/ensemble_based_methods.rst

```diff
@@ -159,9 +159,11 @@ equation for how the boat moves in time, and at selected points in time :math:`t
 :math:`\sigma_k`: The uncertainty in the position at time :math:`t_k`.
 
 :math:`x_k^{\ast}`: The *estimated/forecasted* position at time :math:`t_k`.
-   This is the position estimated from :math:`x_{k-1}` and :math:`g(x,t)`, but
+   This is the position estimated from :math:`x_{k-1}` using :math:`g(x,t)`, but
    before the observed data :math:`d_k` are taken into account.
 
+:math:`\sigma_k^{\ast}`: The uncertainty of estimate / forcast.
+
```

Comment:
Could we be more clear that it belongs together with `x_k^*` by writing out the math-quantity, e.g
"The uncertainty associate estimate / forecast, :math:`x_k^{\ast}`, at time k."